src/odemis/acq/test/calibration_test.py

```diff
@@ -145,15 +145,21 @@ def test_load_simple_arpol(self):
                              model.MD_POL_POS_QWP: linpol_positions[idx],  # rad
                              })
 
+        bg_list = []
         bg_data = []  # list of background images
         for idx in range(len(pol_positions)):
             dcalib = numpy.ones((1, 1, 1, 512, 1024), dtype=numpy.uint16) + idx  # different bg image for each pol
             calib = model.DataArray(dcalib, metadata[idx])
             bg_data.append(calib)
+            bg_list.append(calib.tolist())
 
         # Give one DA, the correct one, so expect to get it back
         out = calibration.get_ar_data(bg_data)
-        numpy.testing.assert_equal(out, bg_data)  # all bg images should have same shape
+        out_list = []
+        for o_data in out:
+            out_list.append(o_data.tolist())
+
+        self.assertEquals(bg_list, out_list, msg="compare lists ignoring the order")
```

Comment:
I don't understand how this works, as this is still a list, so it still has a problem with the order. 
I'd recommend something like:
```python
# Check the expected and return lists contain the same DataArrays
self.assertEqual(len(bg_data), len(out))
for da in out:
    self.assertIn(da, bg_data)
```